cc_utilities/legacy_upload.py

```diff
@@ -405,9 +405,7 @@ def validate_phone_number_field(raw_value, country_code="US"):
         number = phonenumbers.parse(raw_value, country_code)
         return phonenumbers.is_valid_number(number)
     except Exception:
-        import pdb
```

Comment:
oops!

Reply:
yeah I noted this on the commit to somewhere yesterday but it was a commit comment since it wasn't in a pr so not sure the notice went anywhere useful :-)